src/covid19sim/hydra-configs/base_method.yaml

```diff
@@ -0,0 +1,45 @@
+#FORMER COMMAND-LINE ARGS
+n_people: 1000
+init_percent_sick: 0.001
+start_time:
+  "2020-02-28 00:00:00" # in utils.py/parse_configuration, this will be parsed
+  #his will be parsed into a datetime object with the following schema: '%Y-%m-%d %H:%M:%S'
+simulation_days: 30
+outdir: ./output
+out_chunk_size: null
+seed: 0
+n_jobs: 1
+port: 6688
+tune: False
+name: ""
```

Comment:
I will really like to give `name` as a command-line argument. It is because when I run it, I want to give it an identifier that will make it easy for me to identify the output.

Reply:
changing the config file everytime adds one more step to the pipeline.